index.html

```diff
@@ -1,6 +1,244 @@
+<head>
+  <meta charset="utf-8">
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css" integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous">
+
+  <link rel="stylesheet" href="style.css">
+  <title></title>
+</head>
+
+<body>
+  <nav class="main-nav">
+    <ul class="ul">
+      <li><a href="#">Home</a></li>
+      <li><a href="#">About</a></li>
+      <li><a href="#">Contact</a></li>
+      <li><button class="quiz-btn">Take the quiz</button></li>
+    </ul>
+  </nav>
+
+  <main>
+    <section class="quiz">
+      <i class="exit-icon fas fa-times fa-2x"></i>
+      <div class="quiz-flex">
+        <h2 class="quiz-title">Do you have depression?</h2>
+        <div class="quiz-ctn">
+          <h3 class="quiz-question">1. Little interest or pleasure in doing things</h3>
+          <div class="quiz-choices">
+            <label for="not-at-all--1" class="quiz-choice" data-score="1">
+              <input class="input" type="radio" id="not-at-all--1" name="btn-1">
+              Not at all
+            </label>
+            <label for="several-days--1" class="quiz-choice" data-score="2">
+              <input class="input" type="radio" id="several-days--1" name="btn-1">
+              Several days
+            </label>
+            <label for="more-than-half--1" class="quiz-choice" data-score="3">
+              <input class="input" type="radio" id="more-than-half--1" name="btn-1">
+              More than half the days
+            </label>
+            <label for="nearly-everyday--1" class="quiz-choice" data-score="4">
+              <input class="input" type="radio" id="nearly-everyday--1" name="btn-1">
+              Nearly every day
+            </label>
+          </div>
+        </div>
+        <div class="quiz-ctn">
+          <h3 class="quiz-question">2. Feeling down, depressed, or hodiveless</h3>
+          <div class="quiz-choices">
+            <label for="not-at-all--2" class="quiz-choice" data-score="1">
+              <input class="input" type="radio" id="not-at-all--2" name="btn-2">
+              Not at all
+            </label>
+            <label for="several-days--2" class="quiz-choice" data-score="2">
+              <input class="input" type="radio" id="several-days--2" name="btn-2">
+              Several days
+            </label>
+            <label for="more-than-half--2" class="quiz-choice" data-score="3">
+              <input class="input" type="radio" id="more-than-half--2" name="btn-2">
+              More than half the days
+            </label>
+            <label for="nearly-everyday--2" class="quiz-choice" data-score="4">
+              <input class="input" type="radio" id="nearly-everyday--2" name="btn-2">
+              Nearly every day
+            </label>
+          </div>
+        </div>
+        <div class="quiz-ctn">
+          <h3 class="quiz-question">3. Trouble falling or staying asleediv, or sleediving too much</h3>
```

Comment:
I think 'asleediv' and 'sleediving' are typos. The same as 'hodiveless' above.
I think you must have wanted to replace all 'p' into 'div' and accidentally replaced these words: 'asleep', 'sleeping', and 'hopeless'